account_financial_report_qweb/report/abstract_report_xlsx.py

```diff
@@ -3,6 +3,8 @@
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from cStringIO import StringIO
+import xlsxwriter
```

Comment:
@pedrobaeza , you're right on this point.
But it's not the subject of this PR.
I will try to do another PR for that, but I don't know when I could do that.